src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -289,6 +290,9 @@ function getLogoSrcs(changes: TransactionChanges): string[] {
 }
 
 function parseUniswapXOrder({ details, chain, timestamp }: OrderActivity): Activity | undefined {
+  // We currently only support displaying pending activity sent from the current client, so we hide pending activty fetched remotely
```

Comment:
can the comment also explain why we only handle pending activity that's locally stored? something to the effect of we dont handle repolling etc etc 

maybe also add a TODO here to handle pending orders from gql + start polling somehow 🤔